packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -87,8 +87,8 @@ export function FileUploader({
           percentage !== 100
             ? { ...status, percentage, fileState: 'resume' }
             : { ...status, percentage, fileState: 'success' };
-        const addPercentage = [...fileStatusesRef.current];
-        setFileStatuses(addPercentage);
+        const newFileStatuses = [...fileStatusesRef.current];
+        setFileStatuses(newFileStatuses);
```

Comment:
Can you make these changes in other `setFileStatuses` too below?

Reply:
Sure

Reply:
a158fbb